packages/tangle/images/TestBookerMarkerMappings.md

```diff
@@ -0,0 +1,230 @@
+
+### Transaction 1 (create Marker): 
+
+- message 1: 
+    - create sequence ID 1, for now sequence ID of Main = 1
+    - PastMarker.BranchID = MainBranchID
+    - Payload.BranchID = MainBranchID
+
+Note that MainBranchID does not own sequence ID 1, although the PastMarker has sequence ID 1.
+
+![](https://imgur.com/Wsobgv0.png)
+
+### Transaction 2 (create Marker, double spend): 
+
+- message 2 double spends message 1
+- message 1:
+    - create Branch A
+    - Branch A gets sequence ID = 1
+    - PastMarker.BranchID = A
+    - Payload.BranchID = A
+- message 2: 
+    - create Branch B
+    - create sequence ID 2
+    - PastMarker.BranchID = B
+    - Payload.BranchID = B
+
+![](https://imgur.com/9SqyjlY.png)
+
+### Transaction 3 (create Marker, double spend): 
+
+- message 3 double spends message 1
+- message 3 
+    - create Branch C
+    - create sequence ID 3
+    - PastMarker.BranchID = C
+    - Payload.BranchID = C
+
+![](https://imgur.com/N8xffwD.png)
+
+### Transaction 4 (create Marker): 
+
+- message 4
+    - create sequence ID 4, for now sequence ID of Main = 4
+    - PastMarker.BranchID = Main
+    - Payload.BranchID = Main
+
+![](https://imgur.com/GIJjo0b.png)
+
+### Transaction 5 (create Marker): 
+
+- message 5
+    - PastMarker.BranchID = Main
+    - Payload.BranchID = Main
+
+![](https://imgur.com/YMGAbiF.png)
+
+### Transaction 6 (create Marker): 
+
+- message 6
+    - PastMarker.BranchID = [A], [.] indicates inheritance
+    - Payload.BranchID = Main, since no double spend
+
+![](https://imgur.com/ZtSYkvq.png)
+
+### Transaction 7 (create Marker): 
+
+- message 7
+    - PastMarker.BranchID = [C], [.] indicates indirect mapping of the marker to the Branch
+    - Payload.BranchID = Main, since no double spend
+
+![](https://imgur.com/eqmTCnY.png)
+
+### Transaction 8 (create Marker): 
+
+- message 8
+    - PastMarker.BranchID = [A], [.] indicates indirect mapping of the marker to the Branch, 
+    - Payload.BranchID = Main, since no double spend
+
+![](https://imgur.com/gbLlGgd.png)
+
+### Transaction 9 (create Marker, double spend): 
+
+- message 9
+    - PastMarker.BranchID = [C], [.] indicates indirect mapping of the marker to the Branch
+    - Payload.BranchID = Main, since no double spend
+
+![](https://imgur.com/i6ZFvnF.png)
+
+### Transaction 10 (create Marker, double spend): 
+
+- message 5
+    - create Branch D
+    - change: PastMarker.BranchID = D
+    - change: Payload.BranchID = D
+- message 10
+    - double spends message 5
+    - create Branch E
+    - PastMarker.BranchID = B+E
+    - Payload.BranchID = E
+
+![](https://imgur.com/lImMAQY.png)
+
+### Transaction 11 (create Marker): 
+
+- Message 11: 
+    - PastMarker.BranchID = A+C
+    - Payload.BranchID = Main
+
+![](https://imgur.com/WT0qEEH.png)
+
+### Transaction 12: 
+
+- Message 12: 
+    - no marker !
+    - PastMarker.BranchID = (1,3), (3,3)
+    - Payload.BranchID = Main
+    - MessageMetadata.BranchID = A+C
```

Comment:
Maybe make this bold and/or add something like: "Message needs to be individually mapped to branches."